src/main/java/com/flightstats/hub/model/ChannelConfig.java

```diff
@@ -93,15 +93,21 @@ private ChannelConfig(String name,
             this.ttlDays = ttlDays;
             this.maxItems = maxItems;
         }
-
         if (isBlank(storage)) {
-            this.storage = SINGLE.name();
```

Comment:
While you're in here, can you extract the storage type choice logic into a method that returns the type? Then you'd only have to do `this.storage = getStorageType()` or something similar, which sets storage just once. That'd move the historical tagging back out to where it was, but I think right now it gets lost in all the storage logic.